pyclam/criterion.py

```diff
@@ -392,3 +393,40 @@ def __call__(self, root: Cluster) -> Set[Cluster]:
             else:  # add children to frontier
                 frontier.update(cluster.children)
         return selected
+
+
+class Labels(SelectionCriterion):
+    """ Uses a given dictionary of index -> label to select optimal clusters.
+
+    During a BFT of the tree, when a cluster has an overwhelming majority of points with the same label,
+    that cluster is selected.
+    If some data points are unlabelled, then when a cluster contains no labelled points, it too is selected.
+
+    The 'overwhelming majority' fraction is parameterized by the fraction of the minority class.
+    Let f be the fraction of the smallest minority class in all the data.
+    Then the threshold for determining an overwhelming majority is (1 - f / 10).
```

Comment:
I meant you should parameterize the magical `10`, unless there's a good reason for it to be 10?

Reply:
10 is a very good number here so I am going to choose an appropriate default value for the argument